package.json

```diff
@@ -3288,48 +3287,7 @@
                 }
             }
         ],
-        "walkthroughs": [
```

Comment:
Removes walkthrough since we will be showing the Add Connection page on first startup